GeoLib/Grid.h

```diff
@@ -486,9 +480,11 @@ template <typename POINT>
 void Grid<POINT>::initNumberOfSteps(std::size_t n_per_cell,
 	std::size_t n_pnts, std::array<double,3> const& delta)
 {
+	double const max(* std::max_element(delta.cbegin(), delta.cend()));
+
 	std::bitset<3> dim; // all bits set to zero
 	for (std::size_t k(0); k<3; ++k) {
-		if (std::abs(delta[k]) >= std::numeric_limits<double>::epsilon()) {
+		if (delta[k] >= 1e-4 * max) {
```

Comment:
> What do you mean by documentation on the website?

I confused it. I thought, this PR was about some preprocessing tool. So I thought, there was some wiki page about it.